firefox-ios/Client/Frontend/Browser/TermsOfUse/TermsOfUseStrings.swift

```diff
@@ -0,0 +1,51 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+import Common
+import Localizations
+import Shared
+
+struct TermsOfUseStrings {
+    static let titleText = TermsOfUse.Title
+
+    static var descriptionText: String {
+        return String.localizedStringWithFormat(TermsOfUse.Description, AppName.shortName.rawValue)
+    }
+
+    static let reviewAndAcceptText = TermsOfUse.ReviewAndAcceptText
+    static let acceptButtonTitle = TermsOfUse.AcceptButton
+    static let remindMeLaterButtonTitle = TermsOfUse.RemindMeLaterButton
+
+    static var combinedText: String {
```

Comment:
nit: Just a heads up, this is a combined string, and the naming could be a bit more descriptive.

Reply:
OK, i changed the name. Thank you!